contrib/pyln-client/pyln/client/gossmap.py

```diff
@@ -164,6 +211,15 @@ def __eq__(self, other):
     def __hash__(self):
         return self.scid.__hash__()
 
+    def has_feature(self, bit):
+        return 1 << bit & self.features > 0
+
+    def has_feature_odd(self, bit):
+        return 2 << bit & self.features > 0
+
+    def has_features(self, bitmask):
+        return bitmask & self.features == bitmask
```

Comment:
I don't think these functions are generally useful.  Most commonly, you want all nodes which support feature_foo, you want "features & (3 << foo) != 0".  Call that "has_feature()".  Then you can have "has_compulsory_feature()" and "has_optional_feature()".

has_features() should almost certainly use the "has_feature" logic, too.

Reply:
Please fix this.  This API is terrible!

Reply:
Yes I did that now, can you re-check?